chart/templates/workers/worker-deployment.yaml

```diff
@@ -247,6 +249,7 @@ spec:
         - name: worker-log-groomer
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          securityContext: {{ $containerSecurityContext | nindent 12 }}
```

Comment:
Same here, `workers.logGroomerSidecar.securityContexts.container`?

Reply:
good point. I went through the sidecars and added the helper with the config from the values.yaml